### ISSUE TITLE
Personalize Your Postprocessing

### ISSUE THREAD

**Issue by evan-erdos** [briswalsh/SpringCleaning#162]
Now that you've got it working, do something creative with the effects to make your scene pop!
Brighter! Darker! Moodier! Less Moody! Purple shadows! Blue Greens!
Email a screenshot of your favorite thing and send it to your TA so he can brag to other students about the best environment the class has yet seen!

**Comment by bfactor** [comment]
Yeah....so your lighting changes weren't baked in. You don't want to see it now. Trust me. Damage control here we go!

**Comment by evan-erdos** [comment]
ok, remove the huge spotlights you had, there's other ways to increase light (ambient light under lighting palette). 
If you don't remove them you risk fooling up your lighting, or failing that, you'll just be creating weird shadows that don't make sense.

The directional light is all you need. Send a screenshot so I can tell you what's wrong with your light baking settings.

Given that your performance is now drastically improved, you could even get away with just running lighting in realtime

**Comment by bfactor** [comment]
I think we fixed it...what do you think?

![ben1](https://user-images.githubusercontent.com/23264845/32129749-151e8e7c-bb59-11e7-943c-10424426d639.PNG)
![ben2](https://user-images.githubusercontent.com/23264845/32129750-17124f8e-bb59-11e7-9881-a364f49af17e.PNG)

**Comment by evan-erdos** [comment]
so you turned off all the postprocessing? that's your solution?

**Comment by briswalsh** [comment]
Shucks, you figured it out. It might come back if we have time to talk on Monday.

**Comment by bfactor** [comment]
Shucks, you figured it out. It might come back if we have time to talk on Monday.

**Comment by evan-erdos** [comment]
Turn off the processing if you want, but you'd do well to keep the changes I made.
The emissive on many materials was way too high, and you had an unreasonable number of realtime lights.
At least turn off the three enormous spotlights, as I wholeheartedly suspect their unreasonably high ranges are also causing your realtime shadow resolution problem, not to mention they're obscuring whatever nice shadows you might be getting from the directional light. 
In Forward Rendering, every extra realtime pixel light has an outrageous cost. 
That's where the "10 million" polygons came from, despite there only being 2 million polygons in the scene.

As a team you guys seem to have developed more superstitions than game mechanics.
You're clinging to it because it "works", but you're rendering everything 3 extra times and it's probably why your shadows are broken.
I don't understand how you came to be this traumatized, but you can't let Unity boss you around.

tl;dr; your lighting setup is untenable and needs to be fixed regardless, make time for it
